cmd/license.go

```diff
@@ -43,61 +42,54 @@ func extractLicenseIDs() error {
 		return err
 	}
 
-	// create two slices of license IDs, one for deprecated and one for not deprecated
+	// create two slices of license IDs, one for deprecated and one for active
+	var activeLicenseIDs []string
```

Comment:
Prefer `activeLicenseIDs` instead of `nonDeprecatedLicenseIDs` as it is more descriptive.